extensions/wikia/AdEngine/js/lookup/prebid/prebidHelper.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.adEngine.lookup.prebid.prebidHelper', function() {
+	var adUnits = [];
```

Comment:
this variable is used in two methods inside this module. I can remove one of those modules and move its logic to `forEach` loop. Although I it may decrease readability. I'm fine both ways.